bot/cogs/snakes.py

```diff
@@ -16,32 +19,88 @@ def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
     async def get_snek(self, name: str = None) -> Dict[str, Any]:
-        """
-        Go online and fetch information about a snake
+        with open('bot/db/snakes.json', 'r') as file:
+                snakes_dict = json.load(file)
 
-        The information includes the name of the snake, a picture of the snake, and various other pieces of info.
-        What information you get for the snake is up to you. Be creative!
+        if name is None or len(name) == 0:
+            _, snake_info = random.choice(list(snakes_dict.items()))
 
-        If "python" is given as the snake name, you should return information about the programming language, but with
-        all the information you'd provide for a real snake. Try to have some fun with this!
+        elif len(name) > 0:
+            snake = snakes_dict[name.lower()]
+            if snake['name'] != "python":
+                snake_info = {
+                    'name': snake['name'],
+                    'description': snake['description'],
+                    'location': snake['location'],
+                    'venomous': snake['venomous'],
+                    'image': snake['image']
+                }
```

Comment:
If this is how the information is stored, then you could just `return snake`, there's no need to repackage the dict into another dict.

Reply:
I see what you mean, I think I did that because I have different information for the Python Language. I guess I would only need to change `snake` if they entered `python` if I'm correct.

Reply:
Well, no, I mean, the keys you're placing into the dict match the keys that you're pulling out of the snake dict. It's just redundant.